packaging/installer/kickstart.sh

```diff
@@ -471,12 +471,20 @@ cd netdata-* || fatal "Cannot cd to netdata source tree"
 # ---------------------------------------------------------------------------------------------------------------------
 # install netdata from source
 
-if [ -x netdata-installer.sh ]; then
+install() {
   progress "Installing netdata..."
   run ${sudo} ./netdata-installer.sh ${NETDATA_UPDATES} ${NETDATA_INSTALLER_OPTIONS} "${@}" || fatal "netdata-installer.sh exited with error"
   if [ -d "${ndtmpdir}" ] && [ ! "${ndtmpdir}" = "/" ]; then
     run ${sudo} rm -rf "${ndtmpdir}" > /dev/null 2>&1
   fi
+}
+
+if [ -x netdata-installer.sh ]; then
+  install "$@"
 else
-  fatal "Cannot install netdata from source (the source directory does not include netdata-installer.sh). Leaving all files in ${ndtmpdir}"
+  if [ "$(find . -mindepth 1 -maxdepth 1 -type d | wc -l)" -eq 1 ] && [ -x "$(find . -mindepth 1 -maxdepth 1 -type d)/netdata-installer.sh" ]; then
```

Comment:
@Ferroin Just to better understand how this should work.

Why is the line count check in the if condition required? Isn't existence and "executability" of the sh script, in combination with **find . -mindepth 1 -maxdepth 1 -type d** as script's parent directory, a sufficient check? 

If there are more than one parent directories for the script the condition will fail.

Is there some case that 
`find . -mindepth 1 -maxdepth 1 -type d/netdata-installer.sh` would be executed while `"$(find . -mindepth 1 -maxdepth 1 -type d | wc -l)" -eq 1` would fail? :thinking_face:

Reply:
Find returns successfully as long as it finds any directories, while the case we are trying to account for here is a particular bug that causes our sources to be extracted into a sub-directory of the temporary working directory instead of directly into the temporary working directory. I’ve intentionally made the condition to check for this case as specific as possible to minimize the potential for this introducing other bugs.